tests_v2/test_focus_history_shortcut_cross_workspace.py

```
@@ -0,0 +1,120 @@
+            return current
+        time.sleep(0.1)
+    return current
+
+
+def _wait_selected(c: cmux, wsid: str, timeout: float = 5.0) -> None:
+    deadline = time.time() + timeout
+    while time.time() < deadline:
+        if _selected_workspace(c) == wsid:
+            return
+        time.sleep(0.1)
+    raise cmuxError(f"workspace {wsid} never became selected")
```

Ruff (0.16.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)